_translator-files/po/sv-SE/Client-Troubleshooting.po

```diff
@@ -70,276 +71,278 @@ msgid ""
 "* TOC\n"
 " {:toc}\n"
 msgstr ""
+"* TOC\n"
+" {:toc}\n"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:19
 #, markdown-text, no-wrap
 msgid "</details>\n"
-msgstr ""
+msgstr "</details>\n"
 
 #. type: Title ###
 #: ../wiki/en/Client-Troubleshooting.md:20
 #, markdown-text, no-wrap
 msgid "Don’t hear any sound/Others don’t hear you?"
-msgstr ""
+msgstr "Hör inget ljud/andra hör dig inte?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:23
 #, markdown-text
 msgid "Start with the simple stuff: make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your sound card. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://github.com/jamulussoftware/jamulus/discussions) for guidance."
-msgstr ""
+msgstr "Börja med de enkla sakerna: se till att ditt instrument/mikrofon och hörlurar är anslutna till rätt uttag. Se till att inga andra applikationer som din webbläsare, videokonferensverktyg etc. också använder ditt ljudkort. Du bör stänga av dem när du använder Jamulus. Om allt ser OK ut och problemet kvarstår, är det troligtvis ett problem med inställningarna för din ljudenhet. Detta beror på din specifika konfiguration (plattform, hårdvara, mjukvara och drivrutiner), så det är bäst att fråga på [forumen](https://github.com/jamulussoftware/jamulus/discussions) för vägledning."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:25
 #, markdown-text, no-wrap
 msgid "**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4ALL setup section](Installation-for-Windows#setting-up-asio4all)\n"
-msgstr ""
+msgstr "**Windows-användare (ASIO4ALL)**: Om du använder ASIO4ALL-drivrutinen ta en titt på [ASIO4ALL-installationsavsnittet](Installation-for-Windows#setting-up-asio4all)\n"
```

Comment:
Text in link after # should be translated - with text that matches the corresponding paragraph title.

Reply:
@genesisproject2020 could you please fix those issues - either on weblate or in a separate PR to this repo?